src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -332,6 +332,11 @@ function MoneyRequestView({
     const noticeTypeViolations = transactionViolations?.filter((violation) => violation.type === 'notice').map((v) => ViolationsUtils.getViolationTranslation(v, translate)) ?? [];
     const shouldShowNotesViolations = !isReceiptBeingScanned && canUseViolations && ReportUtils.isPaidGroupPolicy(report);
 
+    const errors = {
```

Comment:
@kaushiktd 
I missed. Why did we create this errors object if we are not going to use it?
In the proposal we mentioned to use this errors object?

Reply:
@abdulrahuman5196 On the existing report thread page, only errors from the transaction object `transaction?.errors` are shown. However, there are instances where the `parentReportAction` variable also contains errors. Therefore, we need to create an error object and assign both errors to it.

In the [proposal](https://github.com/Expensify/App/issues/39689#issuecomment-2040156570), I included an additional if statement for the error object. However, when we submitted the PR, you asked to remove the if statement [here](https://github.com/Expensify/App/pull/40460#discussion_r1581393649).

Reply:
@kaushiktd the question is different, we are not using this error object to show the errors? And Why is that?

Reply:
you have created the `errors` object but not using the `errors` to show the specific error?

Reply:
You need to update line# 348 with 
errors={errors} as I mentioned in my proposal.

In proposal line 266 has mentioned before any new merger